### PR TITLE
Upgrade to future

### PR DESCRIPTION
**Pull request by mrmlnc**
Changelog:
  * Update dependencies
  * Use `Promise.denodeify` instead `promisify-node`
  * Pretty code
  * Update Travis CI
  * Added badges for the [David](https://david-dm.org/just-boris/less-plugin-glob), npm and Travis CI status

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
+var path = require('path');
+var eol = require('os').EOL;
+var globbyPromise = Promise.denodeify(globby);
 
 function processPaths(paths) {
-    return paths.filter(function(path) {
-        if(!isLess(path)) {
-            console.warn('Here is non-less file: ' + path + ', ignored');
+    return paths.filter(function(filepath) {
+        if (path.extname(filepath) !== '.less') {
```
Comment (just-boris) [inline index.js]: Why do you think that is nice to inline this function?
I considered to have a separate function with a proper description

**Comment by just-boris** [comment]
Thanks for this PR! I makes the package really nice.
Please, return back the function `isLess`, and I am going to merge this then

File: index.js (hunk 1 above)
Comment (mrmlnc) [inline index.js]: Sorry, I don't know why I did it :hammer:

**Comment by just-boris** [comment]
Great work, thanks

**Comment by just-boris** [comment]
Published as `1.1.1`

**Pull request merged**